src/libs/OptionsListUtils.js

```diff
@@ -368,18 +375,32 @@ function getNewGroupOptions(
  * Build the options for the Sidebar a.k.a. LHN
  * @param {Object} reports
  * @param {Object} personalDetails
- * @param {Obejct} draftComments
+ * @param {Object} draftComments
  * @param {Number} activeReportID
+ * @param {String} priorityMode
  * @returns {Object}
  */
-function getSidebarOptions(reports, personalDetails, draftComments, activeReportID) {
+function getSidebarOptions(reports, personalDetails, draftComments, activeReportID, priorityMode) {
+    let sideBarOptions = {
+        prioritizePinnedReports: true,
+        hideReadReports: false,
+        sortByAlphaAsc: false,
```

Comment:
NAB, `hideReadReports` and `sortByAlphaAsc` are `false` by default so we don't need to be explicit here.